change/@fluentui-react-card-eeb031d6-d039-4293-8d3b-9e84e834670f.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=31953&commitId=58e389e5a3e2a8db0ee156b9317a1484e7f2fcd3&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.badgeMask - RTL.normal.chromium.png|2| Changed|

</details>


  <details><summary><b>Card Converged - Interactive</b> 24 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Card Converged - Interactive.appearance interactive - Filled Alternative - High Contrast.hover.chromium.png|3215| Changed|
|Card Converged - Interactive.appearance interactive - Filled - High Contrast.normal.chromium.png|199| Changed|
|Card Converged - Interactive.appearance interactive - Filled - High Contrast.click.chromium.png|3215| Changed|
|Card Converged - Interactive.appearance interactive - Filled - High Contrast.hover.chromium.png|3215| Changed|
|Card Converged - Interactive.appearance interactive - Filled.normal.chromium.png|171| Changed|
|Card Converged - Interactive.appearance interactive - Filled Alternative.normal.chromium.png|171| Changed|
|Card Converged - Interactive.appearance interactive - Filled Alternative.hover.chromium.png|171| Changed|
|Card Converged - Interactive.appearance interactive - Filled Alternative.click.chromium.png|171| Changed|
|Card Converged - Interactive.appearance interactive - Outline - High Contrast.normal.chromium.png|199| Changed|
|Card Converged - Interactive.appearance interactive - Filled Alternative - High Contrast.normal.chromium.png|199| Changed|
|Card Converged - Interactive.appearance interactive - Outline.normal.chromium.png|171| Changed|
|Card Converged - Interactive.appearance interactive - Outline.click.chromium.png|171| Changed|
|Card Converged - Interactive.appearance interactive - Subtle - High Contrast.click.chromium.png|3215| Changed|
|Card Converged - Interactive.appearance interactive - Subtle.click.chromium.png|171| Changed|
|Card Converged - Interactive.appearance interactive - Outline.hover.chromium.png|171| Changed|
|Card Converged - Interactive.appearance interactive - Subtle.normal.chromium.png|171| Changed|
|Card Converged - Interactive.appearance interactive - Filled Alternative - High Contrast.click.chromium.png|3215| Changed|
|Card Converged - Interactive.appearance interactive - Subtle - High Contrast.normal.chromium.png|199| Changed|
|Card Converged - Interactive.appearance interactive - Filled.click.chromium.png|171| Changed|
|Card Converged - Interactive.appearance interactive - Subtle.hover.chromium.png|171| Changed|
|Card Converged - Interactive.appearance interactive - Subtle - High Contrast.hover.chromium.png|3215| Changed|
|Card Converged - Interactive.appearance interactive - Filled.hover.chromium.png|171| Changed|
|Card Converged - Interactive.appearance interactive - Outline - High Contrast.click.chromium.png|3215| Changed|
|Card Converged - Interactive.appearance interactive - Outline - High Contrast.hover.chromium.png|3215| Changed|

</details>


  <details><summary><b>Card Converged - Selectable</b> 52 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Card Converged - Selectable.appearance selectable - Filled - High Contrast.hover.chromium.png|3215| Changed|
|Card Converged - Selectable.appearance focusable + selectable.hover.chromium.png|171| Changed|
|Card Converged - Selectable.appearance focusable + selectable.normal.chromium.png|171| Changed|
|Card Converged - Selectable.appearance selectable - Filled - High Contrast.click.chromium.png|3215| Changed|
|Card Converged - Selectable.appearance selectable - Filled - RTL.selected.chromium.png|171| Changed|
|Card Converged - Selectable.appearance selectable - Filled - High Contrast.normal.chromium.png|199| Changed|
|Card Converged - Selectable.appearance selectable - Filled - RTL.hover.chromium.png|171| Changed|
|Card Converged - Selectable.appearance selectable - Filled - RTL.normal.chromium.png|171| Changed|
|Card Converged - Selectable.appearance selectable - Filled Alternative - RTL.hover.chromium.png|171| Changed|
|Card Converged - Selectable.appearance selectable - Filled Alternative - RTL.normal.chromium.png|171| Changed|
|Card Converged - Selectable.appearance selectable - Filled - High Contrast.selected.chromium.png|3215| Changed|
|Card Converged - Selectable.appearance selectable - Filled - RTL.click.chromium.png|171| Changed|
|Card Converged - Selectable.appearance selectable - Filled Alternative - High Contrast.hover.chromium.png|3215| Changed|
|Card Converged - Selectable.appearance selectable - Filled Alternative - High Contrast.click.chromium.png|3215| Changed|
|Card Converged - Selectable.appearance selectable - Filled Alternative - High Contrast.selected.chromium.png|3215| Changed|
|Card Converged - Selectable.appearance selectable - Filled Alternative.normal.chromium.png|171| Changed|
|Card Converged - Selectable.appearance selectable - Filled Alternative - High Contrast.normal.chromium.png|199| Changed|
|Card Converged - Selectable.appearance selectable - Filled Alternative.click.chromium.png|171| Changed|
|Card Converged - Selectable.appearance selectable - Filled Alternative.selected.chromium.png|171| Changed|
|Card Converged - Selectable.appearance selectable - Filled.selected.chromium.png|171| Changed|
|Card Converged - Selectable.appearance selectable - Filled.normal.chromium.png|171| Changed|
|Card Converged - Selectable.appearance selectable - Filled.click.chromium.png|171| Changed|
|Card Converged - Selectable.appearance selectable - Filled Alternative - RTL.selected.chromium.png|171| Changed|
|Card Converged - Selectable.appearance selectable - Filled Alternative.hover.chromium.png|171| Changed|
|Card Converged - Selectable.appearance selectable - Filled.hover.chromium.png|171| Changed|
|Card Converged - Selectable.appearance selectable - Outline - High Contrast.click.chromium.png|3215| Changed|
|Card Converged - Selectable.appearance selectable - Outline - RTL.selected.chromium.png|171| Changed|
|Card Converged - Selectable.appearance selectable - Outline - High Contrast.normal.chromium.png|199| Changed|
|Card Converged - Selectable.appearance selectable - Outline - RTL.hover.chromium.png|171| Changed|
|Card Converged - Selectable.appearance selectable - Outline - RTL.click.chromium.png|171| Changed|
|Card Converged - Selectable.appearance selectable - Outline - RTL.normal.chromium.png|171| Changed|
|Card Converged - Selectable.appearance selectable - Outline.click.chromium.png|171| Changed|
|Card Converged - Selectable.appearance selectable - Outline - High Contrast.selected.chromium.png|3215| Changed|
|Card Converged - Selectable.appearance selectable - Outline.selected.chromium.png|171| Changed|
|Card Converged - Selectable.appearance selectable - Subtle - High Contrast.hover.chromium.png|3215| Changed|
|Card Converged - Selectable.appearance selectable - Outline.hover.chromium.png|171| Changed|
|Card Converged - Selectable.appearance selectable - Subtle - High Contrast.click.chromium.png|3215| Changed|
|Card Converged - Selectable.appearance selectable - Subtle - High Contrast.normal.chromium.png|199| Changed|
|Card Converged - Selectable.appearance selectable - Outline.normal.chromium.png|171| Changed|
|Card Converged - Selectable.appearance selectable - Subtle - RTL.normal.chromium.png|171| Changed|
|Card Converged - Selectable.appearance selectable - Subtle - RTL.selected.chromium.png|171| Changed|
|Card Converged - Selectable.appearance selectable - Subtle - High Contrast.selected.chromium.png|3215| Changed|
|Card Converged - Selectable.appearance selectable - Outline - High Contrast.hover.chromium.png|3215| Changed|
|Card Converged - Selectable.appearance selectable - Subtle.click.chromium.png|171| Changed|
|Card Converged - Selectable.appearance selectable - Subtle - RTL.hover.chromium.png|171| Changed|
|Card Converged - Selectable.appearance selectable - Subtle - RTL.click.chromium.png|171| Changed|
|Card Converged - Selectable.appearance selectable - Subtle.hover.chromium.png|171| Changed|
|Card Converged - Selectable.appearance selectable - Subtle.selected.chromium.png|171| Changed|
|Card Converged - Selectable.appearance selectable - Subtle.normal.chromium.png|171| Changed|
|Card Converged - Selectable.appearance focusable + selectable.click.chromium.png|171| Changed|
|Card Converged - Selectable.appearance focusable + selectable.selected.chromium.png|171| Changed|
|Card Converged - Selectable.appearance selectable - Filled Alternative - RTL.click.chromium.png|171| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>

Reply:
Hey @marcosmoura, text on the second button is not visible
![image](https://github.com/microsoft/fluentui/assets/11574680/13cfe919-2b06-4e37-a4f1-4e60f526398a)

Reply:
I was trying to override the main button text color, but for primary buttons it doesn't work. In this case, some buttons will still have a color that doesn't match the contrast ratio, but there's nothing we can do since (from the card scope) we cannot know what kind of content will be placed there and therefore we cannot style these contents.